src/preprocess.ts

```diff
@@ -28,8 +22,8 @@ export function process(
   const isHtmlFile = /\.html$/.test(filePath);
 
   // This is to support angular 2. See https://github.com/kulshekhar/ts-jest/pull/145
-  if (isHtmlFile && jestConfig.globals.__TRANSFORM_HTML__) {
-    src = 'module.exports=`' + src + '`;';
+  if (isHtmlFile && (jestConfig.globals as any).__TRANSFORM_HTML__) {
+    src = 'module.exports=' + JSON.stringify(src) + ';';
```

Comment:
why are we calling json.stringify on something that's already a string?

Reply:
So that the generated js source export a string ;-) 
If you do not stringify it, it'll be like:

```js
module.exports = <div>
  this is my html
</div>;
```

but it's NOT any JSX in there just bare html